src/support/timestamp.c

```diff
@@ -272,12 +272,14 @@ __wt_time_aggregate_validate(
     /*
      * In the case of missing timestamps, we assign the start point to the stop point and newest
      * start durable timestamp may be larger than newest stop timestamp. Check whether start and
-     * stop are equal first.
+     * stop are equal first and then check the newest start durable timestamp against newest stop
```

Comment:
This is confusing, could you elaborate a bit more, the relation between various timestamps and why these comparisons are correct.

Reply:
@bvpvamsikrishna I tried to elaborate on the comments as discussed, but I couldn't break the conditions under each scenario as the conditions and scenarios are negated.